### PR TITLE
Avatar + Badge Branch

### PR DESCRIPTION
**Pull request by tejas-muthya37**
This branch includes the Badge and the Avatar Components.

### CODE REVIEW

**Comment by romabulani** [comment]
Hi Tejas,
Code looks good, but I am not able to see the images for avaars, commit the media folder, so that I can review it.

**Comment by tejas-muthya37** [comment]
@romabulani, doing this right away.

**Comment by tejas-muthya37** [comment]
@romabulani, done. Sorry for not doing that earlier.

**Review by romabulani — APPROVED** [review]
No problem Tejas,
Good work, approved the PR

**Comment by tejas-muthya37** [comment]
@romabulani, highly appreciate it. Thanks for taking the time out :)

File: Component-Page/component-page.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+.demo-div {
+    display: flex;
+    flex-wrap: wrap;
+    background-color: #F7E800;
+    justify-content: center;
+    gap: 1.5rem;
+    align-items: center;
+    color: black;
+    border-radius: 5px;
+    padding: 20px;
```
Comment (Parul1999) [inline Component-Page/component-page.css]: Padding more than 10px can be given as rem.

File: Component-Page/component-page.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    margin-bottom: 1rem;
+}
+
+.code-examples {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border-radius: 5px;
+    width: 100%;
+    background-color: #282c34;
```
Comment (Parul1999) [inline Component-Page/component-page.css]: Use css variables

File: badge.html [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+          <ul>
+            <li><a href="#">Docs</a></li>
+            <li><a href="#">Examples</a></li>
+            <li><a href="#">Reviews</a></li>
+          </ul>
+        </div>
+      </header>
+      <div class="page-body">
+        <div class="page-sidebar">
+          <div class="sidebar-topic">Get Started</div>
```
Comment (Parul1999) [inline badge.html:38]: Give semantic tag

**Comment by Parul1999** [comment]
image
The badges are very far from the text .
Also , go through the code once and use semantic tags , I have commented at one place.
Use CSS Variables for colours.

File: Component-Page/component-page.css (hunk 1 above)
Comment (tejas-muthya37) [reply Component-Page/component-page.css]: Noted. I've updated padding in rem units wherever suitable. for eg. if I have a padding for an element as 10px 12px (top and sides), I'd rather have it as px than convert it into a decimal rem.

File: Component-Page/component-page.css (hunk 2 above)
Comment (tejas-muthya37) [reply Component-Page/component-page.css]: I've done this across the site wherever there were hex values used. Thanks for the suggestion, looks much better now.

File: badge.html (hunk 3 above)
Comment (tejas-muthya37) [reply badge.html:38]: I've replaced the navbar element from div to nav. Replaced the sidebar from div to aside. Thanks for the suggestion.

**Comment by tejas-muthya37** [comment]
@Parul1999 I've incorporated your suggestions into the code. I've brought the badge closer to the text element as well. If these were all the issues, you can take a final look at it again before approving it. If there's anything else, please let me know. Thanks.

**Review by Parul1999 — APPROVED** [review]

**Pull request merged**